src/cwl_platform/sevenbridges_platform.py

```diff
@@ -8,6 +8,37 @@
 
 from .base_platform import Platform
 
+
+class Config:
+    credentials = {
+        'US': {
+            'api_endpoint': 'https://bms-api.sbgenomics.com/v2',
```

Comment:
This is a public repo....hard-coded urls should not be here.  I think this is better passed in from the launcher/code using this repo.

Reply:
Hi @golharam, yes, I see. 
So, we should move credentials (api endpoint, token, profile) from this repo to launcher specific config files on github.com/bmsgh/ repos (RNA, 10X, WES...) and then, in launcher.py, provide those credentials as an input when run PAML code.
Is that correct?

Reply:
yes, I thinks that the best thing to do.

Reply:
credentials, tokens, etc should be provided from the launcher.

Reply:
Hi @golharam,
I moved api endpoint, token and profile from this repo, meaning that those need be provided now as input to detect and connect methods when we call them. If we take RNA launcher.py for example those lines are going to look like this:
`args.platform = PlatformFactory.detect_platform(credentials)`
and
`platform.connect(credentials)`
where **credentials** is a dict (that will be defined in launcher config) with a following structure:
```
credentials = {
            'US': {
                'api_endpoint': '',
                'token': '',
                'profile': ''
            },
            'CN': {
                'api_endpoint': '',
                'token': '',
                'profile': ''
            }
        }
```

Let me know what you think.